expose_test.go

```diff
@@ -0,0 +1,71 @@
+package chromedp
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/chromedp/cdproto/runtime"
+)
+
+func TestExposeFunc(t *testing.T) {
+
+	tests := []struct {
+		name   string
+		Func   BindingFunc
+		param  string
+		result string
+	}{
+		{
+			name: "func1",
+			Func: func(param string) (string, error) {
+				return param + param, nil
+			},
+			param:  "param1",
+			result: "param1param1",
+		},
+		{
+			name: "func2",
+			Func: func(param string) (string, error) {
+				return strings.ToUpper(param), nil
+			},
+			param:  "param2",
+			result: "PARAM2",
+		},
```

Comment:
puppeteer's examples explain the purpose of `exposeFunc` better. Maybe we can copy them as test cases to chromedp. https://github.com/puppeteer/puppeteer/blob/1668d47b2f4d6176438ec1dd3165015181a6fb77/packages/puppeteer-core/src/api/Page.ts#L1160-L1210

Reply:
Thanks for reading and advice。

I think exposeFunc is more like a js bridge implementation，It needs to be injected into the js environment before navigation to ensure that js can be called at any time.  As a function it exists once is enough.

Maybe we need a deeper conversation to make him more reasonable